contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -43,7 +43,7 @@ class Millisatoshi:
     Many JSON API fields are expressed in millisatoshis: these automatically
     get turned into Millisatoshi types. Converts to and from int.
     """
-    def __init__(self, v: Union[int, str, Decimal]):
+    def __init__(self, v: Union[int, str, float, Decimal]):
```

Comment:
Not sure we want `float` anywhere neat monetary amounts. It's a pretty big source of errors in financial software, and asking the user to explicitly round is safer imho.

Reply:
@cdecker Yes I thought that too (first), but on the same topic we can decide to allow or disallow division that rounds down / floors to int:   `8msat / 5   :=  int(1)`. I find @gallizoltan is right in that this is inconsistent.